src/SayMore/UI/ProjectWindow/ProjectWindow.cs

```diff
@@ -61,9 +63,51 @@ private ProjectWindow()
 			ExceptionHandler.AddDelegate(AudioUtils.HandleGlobalNAudioException);
 
 			InitializeComponent();
+			// ReSharper disable once VirtualMemberCallInConstructor
 			_titleFmt = Text;
 			_menuShowMPlayerDebugWindow.Tag = _menuProject.DropDownItems.IndexOf(_menuShowMPlayerDebugWindow);
 			_menuProject.DropDownItems.Remove(_menuShowMPlayerDebugWindow);
+
+			KeyPreview = true;
+			KeyDown += Project_KeyDown;
+		}
+
+		private static Control FindFocusedControl(Control control)
+		{
+			var container = control as IContainerControl;
+			while (container != null)
+			{
+				control = container.ActiveControl;
+				container = control as IContainerControl;
+			}
+			return control;
+		}
+
+		private void Project_KeyDown(object sender, KeyEventArgs e)
+		{
+			var control = FindFocusedControl((Control)sender);
+
+			if (!_keyDownTypes.Contains(control.GetType().Name)) return;
+			if (e.KeyCode != Keys.Return) return;
+
+			// skip multiline text boxes
+			if (control is TextBox txt)
+			{
+				if (txt.Multiline) return;
+			}
+			
+			// prevent the ding
+			e.SuppressKeyPress = true;
+
+			// go to the next control in the tab order
+			var next = GetNextControl(control, true);
+			while (next != null)
+			{
+				if (_keyDownTypes.Contains(next.GetType().Name)) break;
```

Comment:
I think what you want is to use Control.SelectNextControl.
It's not clear to me why these three types of controls are the only legitimate landing sites. For example, if there were a datagridview control next in line, wouldn't it be legit to land there? If this is just to prevent landing inside a container, maybe the logic should be checking to see if the control is an IContainerControl. (Perhaps a comment is needed to say what kinds of controls you're trying to avoid and/or why. Or maybe this needs to be a separate list.)